docs/src/guides/launch.md

```diff
@@ -37,6 +37,89 @@ To completely reset the dev environment:
   zkstack ecosystem init
   ```
 
+### Advanced local setup with ZKsync Gateway
+
+- Stop services from any other previous setup:
+
+  ```bash
+  zkstack dev clean all
+  ```
+
+- Start default ecosystem with default `era` chain config:
+
+  ```bash
+  zkstack ecosystem init --deploy-paymaster --deploy-erc20 \
+    --deploy-ecosystem --l1-rpc-url=http://localhost:8545 \
+    --server-db-url=postgres://postgres:notsecurepassword@localhost:5432 \
+    --server-db-name=zksync_server_localhost_era \
+    --ignore-prerequisites --observability=false \
+    --chain era \
+    --update-submodules false
+  ```
+
+- Create additional `gateway` chain as rollup:
+
+  ```bash
+  zkstack chain create \
+    --chain-name gateway \
+    --chain-id 506 \
+    --prover-mode no-proofs \
+    --wallet-creation localhost \
+    --l1-batch-commit-data-generator-mode rollup \
+    --base-token-address 0x0000000000000000000000000000000000000001 \
+    --base-token-price-nominator 1 \
+    --base-token-price-denominator 1 \
+    --set-as-default false \
+    --evm-emulator false \
+    --ignore-prerequisites --update-submodules false
+  ```
+
+- Initialise `gateway` chain:
+
+  ```bash
+  zkstack chain init \
+    --deploy-paymaster \
+    --l1-rpc-url=http://localhost:8545 \
+    --server-db-url=postgres://postgres:notsecurepassword@localhost:5432 \
+    --server-db-name=zksync_server_localhost_gateway \
+    --chain gateway --update-submodules false
+  ```
+
+- Transform `gateway` chain from rollup to ZKsync Gateway settlement layer:
+
+  ```bash
+  zkstack chain gateway convert-to-gateway --chain gateway --ignore-prerequisites
+  ```
+
+- Start `gateway` chain server:
+
+  ```bash
+  zkstack server --ignore-prerequisites --chain gateway &> ./gateway.log &
```

Comment:
I know it is like this in my scripts, but recently I started outputting things into ./zruns/gateway.log and ./zruns/era.log . For this zruns has to be created

Reply:
fixed in last commit